autotest.pm

```diff
@@ -44,6 +44,9 @@ loadtest is called.
 =cut
 
 sub find_script ($script) {
+    foreach my $wheel (glob Cwd::getcwd . "/*/tests/$script") {
+        return $wheel;
```

Comment:
A loop with an unconditional return?

Reply:
We discussed it in the daily. I'll change it to use if and [0] to access the first element